Examples/Filtering/CompositeFilterExample.cxx

```diff
@@ -122,7 +122,7 @@ class CompositeExampleImageFilter :
   typedef RescaleIntensityImageFilter< ImageType, ImageType >  RescalerType;
 //  Software Guide : EndCodeSnippet
 
-  virtual void GenerateData() override;
+  void GenerateData() override;
```

Comment:
It is great you are doing this for examples too, as examples will be copied many times.